about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Concert Festival Website">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>About</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body>
+<div>
+  <nav>
+    <header class="container-fluid header pt-0 pb-0">
+      <ul class="list-unstyled list-inline text-end header-bar lead fs-6">
+        <li class="list-inline-item"><a href="#" aria-label="Facebook"><i class="header-item bi-facebook"></i></a></li>
+        <li class="list-inline-item"><a href="#" aria-label="Twitter"><i class="header-item bi-twitter"></i></a></li>
+        <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">English</a></li>
+        <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">My Page</a></li>
+        <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">Logout</a></li>
+      </ul>
+    </header>
+    <nav class="bg-light d-md-flex desktop-nav">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="index.html"><img class="pt-1 ms-5" src="./assets/mobile/cflogo.png" width="40" height="40" alt="Concert Festival Logo"></a>
+      </div>
+      <div class="text-center desktop-navigation">
+        <ul class="list-unstyled fw-bold d-md-flex align-items-center gap-4 justify-content-center px-5">
+          <li><a class="text-decoration-none list-inline-item desktop-nav-item" href="about.html">About</a></li>
+          <li><a class="text-decoration-none list-inline-item desktop-nav-item" href="index.html#program">Program</a></li>
```

Comment:
- [ ] For a better user experience, Kindly make sure when the user is navigating on the about page, they can be able to see the `home` link in the navigation bar.